node-services/Blog/controller/db.js

```diff
@@ -0,0 +1,87 @@
+var db = require("../database.js");
+var express = require("express");
+
+
+const dbOperations = {
+
+  getAllArticles: async () => {
+
+    try{
+    const articles = await db.collection('articles').get();
```

Comment:
what's the behaviour when there is large number of rows in table? would it try to fetch all rows (which can be very expensive) or is there a default limit ?

Also indent, use IDE's formatter